### ISSUE TITLE
WARNINGS after update react-scripts to v5

### ISSUE THREAD

**Issue by niltonxp2** [antoniopresto/antd-mask-input#34]
* **I'm submitting a ...**
[x] bug report
[ ] feature request
[ ] question about the decisions made in the repository
[ ] question about how to use this project

* **Summary**
I'm getting some warns after update react-scipts from v4 to v5 

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. StackOverflow, personal fork, etc.)

```
WARNING in ./node_modules/antd-mask-input/build/module/index.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from 'C:\Workspace\harv-Harvest-frontend\node_modules\antd-mask-input\src\index.ts' file: Error: ENOENT: no such file or director
y, open 'C:\Workspace\harv-Harvest-frontend\node_modules\antd-mask-input\src\index.ts'
 @ ./src/components/CustomInput/customInput.js 5:0-42 350:37-48
 @ ./src/components/index.js 3:0-52 26:0-368
 @ ./src/NextApp.js 16:0-69 32:39-52 49:42-64
 @ ./src/index.js 5:0-32 16:7-14

WARNING in ./node_modules/antd-mask-input/build/module/lib/MaskedInput.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from 'C:\Workspace\harv-Harvest-frontend\node_modules\antd-mask-input\src\lib\MaskedInput.tsx' file: Error: ENOENT: no such file 
or directory, open 'C:\Workspace\harv-Harvest-frontend\node_modules\antd-mask-input\src\lib\MaskedInput.tsx'
 @ ./node_modules/antd-mask-input/build/module/index.js 1:0-38 2:25-30 3:15-20
 @ ./src/components/CustomInput/customInput.js 5:0-42 350:37-48
 @ ./src/components/index.js 3:0-52 26:0-368
 @ ./src/NextApp.js 16:0-69 32:39-52 49:42-64
 @ ./src/index.js 5:0-32 16:7-14

WARNING in ./node_modules/antd-mask-input/build/module/lib/Pattern.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from 'C:\Workspace\harv-Harvest-frontend\node_modules\antd-mask-input\src\lib\Pattern.ts' file: Error: ENOENT: no such file or di
rectory, open 'C:\Workspace\harv-Harvest-frontend\node_modules\antd-mask-input\src\lib\Pattern.ts'
 @ ./node_modules/antd-mask-input/build/module/lib/inputmask-core.js 5:0-36 54:25-32 393:20-27
 @ ./node_modules/antd-mask-input/build/module/lib/MaskedInput.js 11:0-41 217:27-36
 @ ./node_modules/antd-mask-input/build/module/index.js 1:0-38 2:25-30 3:15-20
 @ ./src/components/CustomInput/customInput.js 5:0-42 350:37-48
 @ ./src/components/index.js 3:0-52 26:0-368
 @ ./src/NextApp.js 16:0-69 32:39-52 49:42-64
 @ ./src/index.js 5:0-32 16:7-14

WARNING in ./node_modules/antd-mask-input/build/module/lib/helpers.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from 'C:\Workspace\harv-Harvest-frontend\node_modules\antd-mask-input\src\lib\helpers.ts' file: Error: ENOENT: no such file or di
rectory, open 'C:\Workspace\harv-Harvest-frontend\node_modules\antd-mask-input\src\lib\helpers.ts'
 @ ./node_modules/antd-mask-input/build/module/lib/MaskedInput.js 12:0-71 63:10-16 77:17-23 240:34-46 245:6-18 304:38-50 309:38-50 315:36-48
 @ ./node_modules/antd-mask-input/build/module/index.js 1:0-38 2:25-30 3:15-20
 @ ./src/components/CustomInput/customInput.js 5:0-42 350:37-48
 @ ./src/components/index.js 3:0-52 26:0-368
 @ ./src/NextApp.js 16:0-69 32:39-52 49:42-64
 @ ./src/index.js 5:0-32 16:7-14

WARNING in ./node_modules/antd-mask-input/build/module/lib/inputmask-core.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from 'C:\Workspace\harv-Harvest-frontend\node_modules\antd-mask-input\src\lib\inputmask-core.ts' file: Error: ENOENT: no such fil
e or directory, open 'C:\Workspace\harv-Harvest-frontend\node_modules\antd-mask-input\src\lib\inputmask-core.ts'
 @ ./node_modules/antd-mask-input/build/module/lib/MaskedInput.js 11:0-41 217:27-36
 @ ./node_modules/antd-mask-input/build/module/index.js 1:0-38 2:25-30 3:15-20
 @ ./src/components/CustomInput/customInput.js 5:0-42 350:37-48
 @ ./src/components/index.js 3:0-52 26:0-368
 @ ./src/NextApp.js 16:0-69 32:39-52 49:42-64
 @ ./src/index.js 5:0-32 16:7-14
```

**Comment by antoniopresto** [comment]
in progress

**Comment by fcolnaghi** [comment]
Arruma logo essa porra ai jhow!

**Comment by antoniopresto** [comment]
I just published version 2.0
See examples in tests, readme or storybook: https://github.com/antoniopresto/antd-mask-input/blob/master/stories/input.stories.tsx
